.octactl.yaml

```diff
@@ -0,0 +1,14 @@
+api_host: us-112.octarinesec.com
+api_port: 443
+config_version: 0.11.2
+docker:
+  email: kubecon@octarinesec.com
+  password: V2VsY29tZVRvT2N0YXJpbmU=
+  username: octarinekubecon
```

Comment:
Recognizing that a public use control plane is underway, are you good with these credentials being public in the meantime?

Reply:
yeah it will be good to move these to either run time params or environment variables